### ISSUE TITLE
Critical Balance problem

### ISSUE THREAD

**Issue by alptolga** [z-classic/zclassic#49]
I mined 19 blocks and 15 blocks confirmed. When I check my t address from ''https://classic.zcha.in'' everything is ok but there is an amount problem when I check my wallet. My wallet shows 98 zcl. But it must be 183.75 zcl so where is the confirmed 7 blocks rewards. By the way I pay 0.25 zcl fee per block.
This is my adress
https://classic.zcha.in/accounts/t1dPdyGYnaE6JWXQRBiAuAWmi3yZYzhnLdZ
Amount: 232.75085002 ZCL
4 blocks are still not confirmed
These details are from my wallet.
    "version" : 1000450,
    "protocolversion" : 170002,
    "walletversion" : 60000,
    "balance" : 98.00005379,
    "blocks" : 33395,
    "timeoffset" : 0,
    "connections" : 4,
    "proxy" : "",
    "difficulty" : 11387.48055626,
    "testnet" : false,
    "keypoololdest" : 1480450825,
    "keypoolsize" : 101,
    "paytxfee" : 0.00000000,
    "relayfee" : 0.00001000,
    "errors" : ""

**Comment by alptolga** [comment]
The links that I give are block which I mined but they aren't exist in zcashwallet. As you see their amounts are still in my address when you check the  https://classic.zcha.in/accounts/t1dPdyGYnaE6JWXQRBiAuAWmi3yZYzhnLdZ
Please check them
https://classic.zcha.in/transactions/d2aab21039b6aab3e2a0eff1f4b6132d0fd62ca137d59f7ab9dc2f661f6f521f
https://classic.zcha.in/transactions/218582a89b1fb65206e63189460f76279614a304ece2d5054ad6a69ffb072584
https://classic.zcha.in/transactions/f6877c2a3c5b96a67f1d47e4b81acad97304fda1acd531ada1d697c99d87cbe5
https://classic.zcha.in/transactions/55a4276489ba2bdcfc127cc2670a908b7167e7784d1fa4b1135293f11c13eedd
https://classic.zcha.in/transactions/781d1398ee44b279f80346091c85f9a84649efedcb67168a72816cf1da569007
https://classic.zcha.in/transactions/b52952e64a9ee22c83a05ae69c7153531f769bceabd69687a62f25e6c1c0a141
https://classic.zcha.in/transactions/44972029e19d850ab3623e1f03eb5f832460ca07cee00ffe27a09a1b685cd7af

**Comment by aayanl** [comment]
They were orphaned, zchain should start showing the correct chain now

**Comment by alptolga** [comment]
Thanks for your answer. We talked about the issue in slack as you remember. I hope you remember you told me  that I mined these blocks in different chain and I told you so how the transactions send which use these blocks for sending and your answer is if there are two or more chain, transactions make to all these chains.
 So it is possible to send transactions from bad chain to right chain too.  I is right?
